### ISSUE TITLE
Smart Typography: in GFM should not be recognized

### ISSUE THREAD

**Issue by chrishrb** [chrishrb/go-grip#14]
Smart Typography: in GFM should not be recognized

(c) (C) copyright
(r) (R) registered
(tm) (TM) trademark

(from https://github.com/chrishrb/go-grip/issues/5)

**Comment by chrishrb** [comment]
do you mean here something like this: https://fletcher.github.io/MultiMarkdown-5/typography.html @Disonantemus
for me these symbols aren't rendered and this is the expected behaviour?

**Comment by Disonantemus** [comment]
Yes, GitHub preview does not support this, I did test with last `go-grip` and is working as expected.
